src/Servers/build.cmd

```diff
@@ -1,3 +1,3 @@
 @ECHO OFF
 SET RepoRoot=%~dp0..\..
-%RepoRoot%\build.cmd -projects %~dp0**\*.*proj %*
+%RepoRoot%\build.cmd -projects %~dp0**\*.csproj %*
```

Comment:
What's the purpose of this change?

Reply:
The `-projects` list isn't split between native and non-native projects. That means `**\*.*proj` would be passed to `dotnet msbuild` and blow up spectacularly because only desktop `msbuild` knows how to build the native projects. Further, because this does **not** pass `-noBuildNative` the native projects are always built.